content/docs/user-guide/how-to/resolve-merge-conflicts.md

```diff
@@ -82,16 +82,15 @@ versions, then you can follow this process:
 
 ### Append-only directories
 
-If you have an "append-only" dataset, where people only add new
-files/directories, DVC provides a so-called
-[merge-driver](https://git-scm.com/docs/git-merge#Documentation/git-merge.txt-mergeltdrivergtname)
-that can automatically resolve Git conflicts for you. To use it, first set it up
-in your Git repo:
+If you have an "append-only" dataset (where people only add new
+files/directories) DVC provides a [Git merge driver] that can automatically
+resolve merge conflicts into a parent branch. To use it, first set it up in your
+Git repo:
```

Comment:
@yathomasi did I express this correctly? Rel https://github.com/iterative/dvc.org/issues/3915#issuecomment-1234126026.

Reply:
It looks like I was mistaken. https://github.com/iterative/dvc.org/issues/3915#issuecomment-1241890681

Reply:
NP, rolling back:
```suggestion
If you have an "append-only" dataset (where people only add new
files/directories) DVC provides a [Git merge driver] that can automatically
resolve merge conflicts for you. To use it, first set it up in your Git repo:
```